addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/HeartBleedActiveScanRule.java

```diff
@@ -64,7 +64,7 @@ public class HeartBleedActiveScanRule extends AbstractHostPlugin {
                         CommonAlertTag.OWASP_2021_A06_VULN_COMP,
                         CommonAlertTag.OWASP_2017_A09_VULN_COMP,
                         CommonAlertTag.WSTG_V42_CRYP_01_TLS));
-        ALERT_TAGS.put(CVE, "");
+        ALERT_TAGS.put(CVE, CommonAlertTag.linkifyCve(CVE));
```

Comment:
Better with e.g. `CommonAlertTag.putCve(ALERT_TAGS, CVE);` to enforce that the key and value match.

Reply:
The first param being the existing map?

Reply:
Yes.